data/json/itemgroups/defense_mode.json

```diff
@@ -162,5 +663,239 @@
       { "item": "UPS_off" },
       { "item": "mininuke" }
     ]
+  },
+  {
+    "type": "item_group",
+    "id": "defense_caravan_bionics",
```

Comment:
Does defense mode have an Autodoc available to install the bionics, or are they auto-installed on purchase?

Reply:
I'll have to check, but most don't have an Autodoc available. And the ones that might are too big to actually search entirely, without the hardcoded limits stopping you.
Could always use the manual bionic installation mod, although that isn't the best workaround.